src/main/liquibase/sql-scripts/0010-FIT-3-create_weights_table.sql

```diff
@@ -0,0 +1,8 @@
+--liquibase formatted sql
+--changeset Piotr Michalak:001-FIT-3-create_weights_table
+CREATE SCHEMA fittracker;
+CREATE TABLE fittracker.weights (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
Good one 👍  bigserial seems more appropriate than bigint, as we won't use negative values.